### ISSUE TITLE
The access token is from the wrong issuer

### ISSUE THREAD

**Issue by Dashue** [bradygaster/azure-tools-vscode#44]
Error during resource group creation:
Error: The access token is from the wrong issuer 'https://sts.windows.net/f8cdef31-a31e-4b4a-93e4-5f571e91255a/'. It must match the tenant 'https://sts.windows.net/88d52eb9-4040-4847-b7f2-0d5eeef9cf39/' associated with this subscription. Please use the authority (URL) 'https://login.windows.net/88d52eb9-4040-4847-b7f2-0d5eeef9cf39' to get the token. Note, if the subscription is transferred to another tenant there is no impact to the services, but information about new tenant could take time to propagate (up to an hour). If you just transferred your subscription and see this error message, please try back later.

Any idea what's causing this?

**Comment by chrisdias** [comment]
i'm seeing the same thing

**Comment by rspaulino** [comment]
I can confirm I'm having the same issue as well. I was trying to deploy a ARM.

**Comment by verylongword** [comment]
I'm having the same problem: 
The access token is from the wrong issuer 'https://sts.windows.net/f8cdef31-a31e-4b4a-93e4-5f571e91....
Maybe interesting that my "wrong issuer" has exactly the same  ID:

Where does that ID come from then ???  Is it a developers test ID that has been left in production code by mistake?  Surely I should not be using the same ID as everyone else !!

Also my attempt to set azure.tenantId as per https://marketplace.visualstudio.com/items?itemName=bradygaster.azuretoolsforvscode did not fix the problem, though I may have done it wrong.

**Comment by bradygaster** [comment]
@chrisdias and @lostintangent - do you guys think implementing az-login here would resolve this issue? if so i'll prioritize getting that done next.

**Comment by lostintangent** [comment]
@bradygaster Yeah, this appears to be the same issue you ran into with az-login, what has since been [resolved](https://github.com/lostintangent/az-login/pull/11).

**Comment by bradygaster** [comment]
thanks so much @lostintangent i'll get on implementing this right away in the product so we're all in the same party.
